.github/workflows/test.yml

```diff
@@ -0,0 +1,17 @@
+name: Packaging consistency check
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.8'
+    - name: Install dependencies
+      run: pip install pyaml jsonschema
```

Comment:
Do we need pyaml instead of PyYAML?

Reply:
Looks like pyaml is built on top of PyYAML, we can use PyYAML instead.